### PR TITLE
docs(notification): up-4432 add swagger doc for /v3/notification

### PR DESCRIPTION
**Pull request by alimran**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2079,6 +2079,79 @@ paths:
...
+          value: Bearer {{accessToken}}
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: >
+          information of a device to be registered.
+        required: true
+        content:
+          application/json:
```
Comment (dreamxchange) [inline swagger.yaml:2103]: To improve doc, please add more validation such as max length or enum for provider.

File: swagger.yaml [hunk 2]
```diff
@@ -2079,6 +2079,79 @@ paths:
...
+      description: >
+        Unregister a device from a notification service provider.
+      parameters:
+        - name: Authorization
+          in: header
+          value: Bearer {{accessToken}}
+          required: true
+          schema:
+            type: string
+      requestBody:
```
Comment (dreamxchange) [inline swagger.yaml:2141]: Why do we need this one?

**Review by dreamxchange — CHANGES_REQUESTED** [review]

File: swagger.yaml [hunk 3]
```diff
@@ -2079,6 +2079,79 @@ paths:
           $ref: '#/components/responses/SuccessResponse'
         '400000':
           $ref: '#/components/responses/BadRequestError'
+  /v2/notification:
```
[the remark this answers is not in the document]
Comment (alimran) [reply swagger.yaml]: changed to v3

File: swagger.yaml (hunk 1 above)
Comment (alimran) [reply swagger.yaml:2103]: validation provided using enum

File: swagger.yaml (hunk 2 above)
Comment (alimran) [reply swagger.yaml]: removed the unused segment "requestBody"

**Review by dreamxchange — APPROVED** [review]

**Pull request merged**